IS0501Test.py

```diff
@@ -802,6 +803,40 @@ def test_40(self, test):
             else:
                 return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_41(self, test):
+        """SDP transport files pass SDPoker tests"""
+
+        api = self.apis[CONN_API_KEY]
+        rtp_senders = []
+        if self.is05_utils.compare_api_version(api["version"], "v1.1") >= 0:
+            # Find all RTP senders for v1.1+
+            for sender in self.senders:
+                url = "single/senders/{}/transporttype".format(sender)
+                valid, response = self.is05_utils.checkCleanRequestJSON("GET", url)
+                if valid:
+                    if response == "urn:x-nmos:transport:rtp":
+                        rtp_senders.append(sender)
+                else:
+                    return test.FAIL("Unexpected response from transporttype resource for Sender {}".format(sender))
+        else:
+            # RTP is the only transport type for v1.0
+            rtp_senders = self.senders
+
+        if len(rtp_senders) == 0:
+            return test.UNCLEAR("Not tested. No resources found.")
+
+        for sender in rtp_senders:
+            path = "single/senders/{}/transportfile".format(sender)
+            try:
+                subprocess.check_output(["sdpoker", "--nmos", "false", self.url + path],

```

Comment:
Should the `--shaping true` command line arg be added to check for presence of "TP" format parameter required by ST 2110-21 Section 8.1?

Reply:
I hadn't added any 2110-specific flags as yet given the potential for plain RTP SDP files to be present. Having said that, as noted in the PR notes I don't know how well sdpoker handles non-2110 files. If it doesn't handle them well then some initial checks for mandatory 2110 parameters may be necessary before analysing each file.

Reply:
> given the potential for plain RTP SDP files to be present

Oh, OK. Is that true for many IS-04/IS-05 implementations, do you think?

Reply:
It's certainly true for ours, and I'd hope it'd become increasingly so elsewhere.